src/client/client.ts

```diff
@@ -213,6 +219,15 @@ export class Client {
     return new Schedules(this.http);
   }
 
+  /**
+   * Access the queue API.
+   * 
+   * Create, read, update or delete queues.
+   */
+  public get queues(): Queues {
```

Comment:
Sorry for the nitpicking can we rename `queues` to queue and `Queues` class to `Queue` 😄

Reply:
Yeahh, I was thinking we want to keep it consistent with `Schedules` but I agree `Queues` is weird